test/feature/test_defmo.py

```diff
@@ -9,31 +9,31 @@
 
 class TestDeFMO:
     @staticmethod
-    def test_shape(self, device, dtype):
+    def test_shape(selfarg, device, dtype):
         inp = torch.ones(1, 6, 240, 320, device=device, dtype=dtype)
         defmo = DeFMO().to(device, dtype)
         defmo.eval()  # batchnorm with size 1 is not allowed in train mode
         out = defmo(inp)
         assert out.shape == (1, 24, 4, 240, 320)
 
     @staticmethod
-    def test_shape_batch(self, device):
+    def test_shape_batch(selfarg, device):
         inp = torch.ones(16, 6, 240, 320, device=device)
         defmo = DeFMO().to(device)
         out = defmo(inp)
         assert out.shape == (16, 24, 4, 240, 320)
 
     @staticmethod
     @pytest.mark.skip("jacobian not well computed")
-    def test_gradcheck(self, device):
+    def test_gradcheck(selfarg, device):
```

Comment:
Replace
>def test_gradcheck(selfarg, device): 
with 
def test_gradcheck(self, device, dtype):

everywhere. That is why tests are failing